src/features/user/MFV2/InfoAndCTA/InfoAndCta.module.scss

```diff
@@ -86,4 +92,82 @@
         max-height: 60px;
         max-width: 60px;
      }
-}
\ No newline at end of file
+}
+
+.donorCircleInvitationHeader {
+    display: flex;
+    justify-content: space-between;
+
+    .headerTitle {
+        font-size: 22px;
+        font-weight: 600;
+    }
+}
+
+.inviationCardsContainer {
+    display: flex;
+    gap: 6px;
+    width: 100%;
+    justify-content: center;
+}
+
+.singleInvitationCardContainer {
+    flex:1;
+    img {
+        width: 100%;
+        transition: transform 0.2s ease-in-out;
+    }
+    img:hover {
+        transform: scale(1.02);
+    }
+}
+
+.singleInvitationCardTitle {
+    font-size: 18px;
+    font-weight: 700;
+    color: #333;
+    max-width: 250px;
+
+    @media screen and (max-width: 900px) {
+        font-size: 16px;
+    } 
+}
+
+.singleInvitationCardText {
+    font-size: 14px;
+    color: #333;
+    max-width: 250px;
+
+    @media screen and (max-width: 900px) {
+        font-size: 12px;
+    } 
+}
+
+@media screen and (max-width: 700px) {
+
+    .inviationCardsContainer {
+        flex-direction: column;
+        gap: 10px;
+    }
```

Comment:
I wonder if we could use flex-wrap and a max-width for the cards to achieve a more responsive layout. When we switch to a column layout at just under 700px, with full width for each card, the cards are a bit too large. This would make adding a card easier as well.

Reply:
This was done because it is crucial to keep all the cards in the same horizontal line according to Ganga, and after 700px the design seems to clustered. If I add a max-width and flex-wrap, the cards will start moving to next line individually
<img width="300" alt="image" src="https://github.com/Plant-for-the-Planet-org/planet-webapp/assets/72646230/787481a7-2909-4275-8b40-51207868bce7">

Reply:
Leave as is in that case.